zarr/storage.py

```diff
@@ -860,8 +860,10 @@ def __setitem__(self, key, value):
         try:
             self._tofile(value, temp_path)
 
-            # move temporary file into place
-            os.replace(temp_path, file_path)
+            # move temporary file into place;
+            # make several attempts at writing the temporary file to get past
+            # potential antivirus file locking issues
+            retry_call(os.replace, (temp_path, file_path))
```

Comment:
@ericgyounkin : just realizing (since I have never been able to test this in anger) doesn't `PermissionError` need listing here? Otherwise, no exceptions are going to be caught.

Reply:
@joshmoore Yes, my mistake.  Just added it.